### PR TITLE
Update repo link

### PR DESCRIPTION
**Pull request by arsdorintbp2003**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/arsdorint.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
  
     {
+      "description": Arsdorint
```
Comment (WilliamDavidHarrison) [inline domains/arsdorint.json:3]:
```suggestion
      "description": "Arsdorint"
```

File: domains/arsdorint.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+      "repo": "https://github.com/arsdorintbp2003/arsodrintbp2003.github.io",
       "owner": {
         "username": "nambp2003",
         "email": "e0851451@u.nus.edu"
       },
       "record": {
-        "CNAME": "hashnode.network"
+        "CNAME": "arsdorint.gitHub.io"
```
Comment (WilliamDavidHarrison) [inline domains/arsdorint.json:10]:
```suggestion
        "CNAME": "arsdorintbp2003.github.io"
```

File: domains/arsdorint.json (hunk 1 above)
Comment (WilliamDavidHarrison) [inline domains/arsdorint.json:3]:
```suggestion
      "description": "Arsdorint"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website

**Comment by arsdorintbp2003** [comment]
Oh sorry, I forgot, my repo is private, have changed to public already

**Comment by WilliamDavidHarrison** [comment]
> Oh sorry, I forgot, my repo is private, have changed to public already

Uhh, you don't have any files?

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Comment by arsdorintbp2003** [comment]
I have updated files in my repo:
image

**Review by wdhdev — CHANGES_REQUESTED** [review]
I still can't seem to access it.

**Comment by arsdorintbp2003** [comment]
This repo: https://github.com/arsdorintbp2003/arsodrintbp2003.github.io

**Comment by WilliamDavidHarrison** [comment]
> This repo: https://github.com/arsdorintbp2003/arsodrintbp2003.github.io

There is a typo in the repo name that's why. You need to change the repo name to `arsdorintbp2003.github.io`.

**Pull request closed without merging**